src/desktop/apps/article/components/__tests__/InfiniteScrollNewsArticle.test.js

```diff
@@ -160,7 +155,6 @@ describe('<InfiniteScrollNewsArticle />', () => {
     const rendered = shallow(<InfiniteScrollNewsArticle {...props} />)
     await rendered.instance().fetchNextArticles()
     rendered.update()
-    rendered.find(RelatedArticlesCanvas).length.should.equal(1)
     rendered.html().should.containEql('More from Artsy Editorial')
```

Comment:
Presence of these components is now tested in reaction, but check for text confirms their presence.